.travis.yml

```diff
@@ -37,6 +37,8 @@ jobs:
     python: 3.6
     script:
     - make types
+    - pip install pytype==2019.5.31
```

Comment:
isn't that part of the makefile?

Reply:
didn't see this comment earlier, sorry
i added this a few weeks ago, back then there was a problem with the pytype version
i'll take it out and see what happens